app/uk/gov/hmrc/tariffclassificationfrontend/controllers/AppealCaseController.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc._
+import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+import uk.gov.hmrc.tariffclassificationfrontend.forms.AppealForm
+import uk.gov.hmrc.tariffclassificationfrontend.models.AppealStatus.AppealStatus
+import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, CaseStatus}
+import uk.gov.hmrc.tariffclassificationfrontend.service.CasesService
+import uk.gov.hmrc.tariffclassificationfrontend.views
+import uk.gov.hmrc.tariffclassificationfrontend.views.CaseDetailPage
+
+import scala.concurrent.Future.successful
+
+@Singleton
+class AppealCaseController @Inject()(authenticatedAction: AuthenticatedAction,
+                                     override val caseService: CasesService,
+                                     override val messagesApi: MessagesApi,
+                                     override implicit val config: AppConfig) extends RenderCaseAction with I18nSupport {
```

Comment:
I18nSupport is already in RenderCaseAction

Reply:
Done